pkg/plugins/golang/v3/scaffolds/internal/templates/readme.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package templates
+
+import "sigs.k8s.io/kubebuilder/v3/pkg/machinery"
+
+var _ machinery.Template = &Readme{}
+
+// Readme scaffolds a README.md file
+type Readme struct {
+	machinery.TemplateMixin
+	machinery.BoilerplateMixin
+}
+
+// SetTemplateDefaults implements file.Template
+func (f *Readme) SetTemplateDefaults() error {
+	if f.Path == "" {
+		f.Path = "README.md"
+	}
+
+	f.TemplateBody = readmeFileTemplate
+
+	return nil
+}
+
+//nolint:lll
+const readmeFileTemplate = `# Project Name
+// TODO(user): Add simple overview of use/purpose
+
+## Description
+// TODO(user): An in-depth paragraph about your project and overview of use
+
+## Getting Started
+In order to use this operator you will need to have a Kubernetes cluster to run against.
+If you need a local cluster for testing you can use [KinD](https://kind.sigs.k8s.io/)
+
+### Running the operator
```

Comment:
```suggestion
### Running on the Cluster
```

Reply:
fixed in commit https://github.com/kubernetes-sigs/kubebuilder/pull/2610/commits/dee7b87ab71c0f0b09e1154c1e1abfaa9732c73c